github/Repository.py

```diff
@@ -2089,6 +2089,21 @@ def create_fork(self, organization=github.GithubObject.NotSet):
         )
         return Repository(self._requester, headers, data, completed=True)
 
+    def transfer(self, new_owner, team_ids=[]):
+        """
+        :calls: `POST /repos/:owner/:repo/transfer <https://developer.github.com/v3/repos/#transfer-a-repository>`_
+        :param new_owner: string or "none" or "*". Required.
+        :param team_ids: list. Optional. Works only under organizations.
+        :rtype: :class:`github.Repository.Repository`
+        """
+        assert new_owner is not None
```

Comment:
``assert isinstance(new_owner, str), new_owner``

Since other objects will also also match the current assert.

Reply:
This isn't required since you assert it's a str just below.